src/main/java/org/commonjava/indy/service/repository/controller/AdminController.java

```diff
@@ -188,11 +195,10 @@ public void delete( final StoreKey key, final String user, final String changelo
             if ( store != null && deleteContent )
             {
                 logger.info( "Delete content of {}", key );
-                //TODO: here should send event to notify store service to delete file contents for this repo
-                //                deleteContent( store );
+                purgeFilesystem( key.toString() );
```

Comment:
I saw the TODO mentioned that it will send event for this, so is there any other actions needed about this deletion ?

Reply:
The following 'storeManager.deleteArtifactStore' will send an event out. But here I just make a direct call to storage service. 
Actually we have a few options. For example, main Indy to catch the event and call the api. To me it looks complicated because the current topic does not fit in terms of properly handling the grouping. In other words, the current config is for each node to handle the event once, which does not fit this case. 
Other is for storage service to catch/handle events. But as of today, the storage service does not deal with events. I want to keep it as is. So I just make a call from repo service to storage service.

Reply:
btw, the main indy handles the 'repo deletion' event in some places. but no one handles the file deletion. this pr fix that.

Reply:
Sounds good, LGTM. thanks for the info.